drivers/pinmux/stm32/pinmux_stm32h7.h

```diff
@@ -13,33 +13,83 @@
  */
 
 /* Port A */
+#define STM32H7_PINMUX_FUNC_PA0_PWM2_CH1 \
+	(STM32_PINMUX_ALT_FUNC_1 | STM32_PUSHPULL_PULLUP)
+#define STM32H7_PINMUX_FUNC_PA0_PWM5_CH1 \
+	(STM32_PINMUX_ALT_FUNC_2 | STM32_PUSHPULL_PULLUP)
 #define STM32H7_PINMUX_FUNC_PA0_USART2_CTS    \
 	(STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_PULLUP)
 #define STM32H7_PINMUX_FUNC_PA0_UART4_TX    \
 	(STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_PULLUP)
 
+#define STM32H7_PINMUX_FUNC_PA1_PWM2_CH2 \
+	(STM32_PINMUX_ALT_FUNC_1 | STM32_PUSHPULL_PULLUP)
+#define STM32H7_PINMUX_FUNC_PA1_PWM5_CH2 \
+	(STM32_PINMUX_ALT_FUNC_2 | STM32_PUSHPULL_PULLUP)
+#define STM32H7_PINMUX_FUNC_PA1_PWM9_CH1N \
```

Comment:
For stm32H742/743, the PA1 pin with Alternate Function 4 is assigned to TIM**15**_CH1N (or PWM**15**_CH1N)   and not PWM9_CH1N

Reply:
@FRASTM Timers 15, 16, 17 are 9, 10 and 11 respectively on all Zephyr DTS files. I don't know the reason (I think it is confusing) but I wanted to keep consistency.

Reply:
@gmarull can you elaborate ?

Reply:
Sorry, my mistake @FRASTM @erwango. It looks like TIM9/10/11 are named 15/16/17 on H7 so that's why I got confused... It's fixed now :-)